templates/components/contextmenucommand.js.sapphire

```diff
@@ -1,25 +1,39 @@
 { "category": "commands" }
 ---
-const { Command } = require('@sapphire/framework')
+const { Command } = require('@sapphire/framework');
+const { ApplicationCommandType } = require('discord-api-types/v9');
```

Comment:
```suggestion
const { ApplicationCommandType } = require('discord-api-types/v10');
```

Reply:
DJS v13 and Sapphire v3 use v9 internally as well

Reply:
https://github.com/sapphiredev/framework/blob/c71b8e58e911ca961add71aeb4d824e113dac3df/src/lib/utils/application-commands/computeDifferences.ts#L1-L15

So that was a lie (in sapphire, in djs v13 its v9....for...some..reason)

Reply:
Lie is the wrong word, just a mistake / something that's incorrect. Lie implies that I intentionally intended to mislead.

Anyway, fixed.